src/classes/Environment.php

```diff
@@ -1821,6 +1821,61 @@ function( $stderr ) {
 		return true;
 	}
 
+	/**
+	 * WP CLI Runner
+	 *
+	 * @param string  $command A WP CLI command.
+	 * @return array
+	 */
+	public function wpCliRunner( $command ) {
+		Log::instance()->write( 'Running wp command: ' . $command, 0 );
+
+		$command = 'wp ' . $command . ' --allow-root';
+
+		$exec_config = new ContainersIdExecPostBody();
+		$exec_config->setTty( true );
+		$exec_config->setAttachStdout( true );
+		$exec_config->setAttachStderr( true );
+		$exec_config->setCmd( [ '/bin/bash', '-c', $command ] );
+
+		$exec_command      = EnvironmentFactory::$docker->containerExec( $this->environment_id . '-wordpress', $exec_config );
+		$exec_id           = $exec_command->getId();
+		$exec_start_config = new ExecIdStartPostBody();
+		$exec_start_config->setDetach( false );
+
+		$stream = EnvironmentFactory::$docker->execStart( $exec_id, $exec_start_config );
+		$output = '';
+		$error = '';
+
+		$stream->onStdout(
+			function( $_stdout ) use ( &$output ) {
+				$output .= $_stdout;
+			}
+		);
+
+		$stream->onStderr(
+			function( $_stderr ) use ( &$error ) {
+				$error .= $_stderr;
+			}
+		);
+
+		$stream->wait();
+
+		$exit_code = EnvironmentFactory::$docker->execInspect( $exec_id )->getExitCode();
+
+		if ( 0 !== $exit_code ) {
+			Log::instance()->write( 'Before script returned a non-zero exit code: ' . $script, 0, 'warning' );
+		}
+
+		$result = [
+			'stdOut' => $output,
+			'stdErr' => $error,
+			'code'   => $exit_code,
+		];
+
+		return $result;
```

Comment:
I'd probably just return the exit code and output the error/output under verbosity like we do in other exec commands.